data/types/base_test.go

```diff
@@ -972,13 +940,6 @@ var _ = Describe("Base", func() {
 				Expect(identityFields).To(BeEmpty())
 			})
 
-			It("returns error if time is empty", func() {
```

Comment:
Should you test the `(*b.Time).IsZero()` check here?

Reply:
good catch, re-added this test.